fastlane/fastlane
# [match] Add Amazon S3 storage.

### Checklist

### Motivation and Context
Adds S3 storage to Match. Closes: https://github.com/fastlane/fastlane/issues/15188

### Description
Uses S3 as a storage mode for `match` credentials.

### Testing Steps
* Add the `aws-sdk-v1` gem to your `Gemfile`
* Run `fastlane match development` command with options: `MATCH_STORAGE_MODE="s3"` , `MATCH_S3_REGION`, `MATCH_S3_ACCESS_KEY`, `MATCH_S3_SECRET_ACCESS_KEY` and `MATCH_S3_BUCKET`
* See match create and upload credentials to S3. 🙂
